{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "1e714b43_bfded2b0",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1001712
      },
      "writtenOn": "2025-07-18T08:02:19Z",
      "side": 1,
      "message": "@kevin.wtmh@gmail.com can you kindly confirm that the logs and described error case match with what you\u0027ve seen ðŸ˜Š",
      "revId": "cf1d0641289381d99cb2e6d9a1c2b3ec197629db",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}